src/components/PriceInputRow.vue

```diff
@@ -59,6 +61,18 @@ export default {
     setCurrencyData(currency) {
       this.priceForm.currency = currency
     },
+    fixComma(input) {
+      return input.replace(/,/g, '.');
+    },
+  },
+  computed: {
+    priceRules() {
+      return [
+        value => !!value && !!value.trim() || 'Price is required',
```

Comment:
needs translations :)

Reply:
I'm not sure I did it right, I didn't see style instructions.